components/next_steps_view/next_steps_view.scss

```diff
@@ -54,22 +55,150 @@
     min-width: 192px;
 }
 
-@media screen and (max-width: 768px) {
-    #app-content.NextStepsView {
-        background-size: auto;
+.NextStepsView__body .Card__header {
+    padding: 1px 0;
+}
+
+.NextStepsView__cardHeader {
+    display: flex;
+    border: none;
+    background: none;
+    padding: 12px;
+    width: 100%;
+
+    > span {
+        margin-left: 14px;
+        font-size: 14px;
+        line-height: 20px;
+        font-weight: 600;
+        color: var(--center-channel-color);
+        align-self: center;
     }
+}
 
-    .NextStepsView__header {
-        padding: 24px 28px;
+.NextStepsView__cardHeaderBadge {
+    width: 26px;
+    height: 26px;
+    background-color: var(--center-channel-color-16);
+    color: var(--center-channel-color);
+    border-radius: 14px;
+    display: flex;
+    justify-content: center;
+    transition: all 200ms ease-in-out;
+
+    span {
+        align-self: center;
+        font-size: 14px;
+        line-height: 14px;
+        font-weight: 600;
     }
-    
-    .NextStepsView__header-headerTopText {
-        font-size: 24px;
-        line-height: 32px;
+}
+
+.NextStepsView__skipGettingStarted {
+    display: flex;
+    justify-content: flex-end;
+    margin-top: 12px;
+
+    button {
+        border: none;
+        background: none;
+        padding: 10px 16px;
+        font-weight: 600;
+        font-size: 12px;
+        line-height: 9px;
+        color: var(--button-bg);
     }
+}
 
-    .NextStepsView__body-main {
-        flex: 1 1 100%;
+.Card {
+    .NextStepsView__button {
+        padding: 12px 20px;
+        line-height: 14px;
+        font-size: 14px;
+        font-weight: 600;
+        border: none;
+
+        &.cancel {
+            background: var(--center-channel-bg);
+            color: var(--button-bg);
+        }
+
+        &.confirm {
+            background: var(--button-bg);
+            border-radius: 4px;
+            color: var(--button-color);
+
+            &.disabled {
+                color: rgba(var(--center-channel-color-rgb), 0.32);
+                background: rgba(var(--center-channel-color-rgb), 0.08);
+            }
+        }
+    }
+    .NextStepsView__wizardButtons {
+        display: flex;
+        justify-content: flex-end;
+        margin: 4px;
+
+        .NextStepsView__button + .NextStepsView__button {
+            margin-left: 8px;
+        }
+    }
+}
+
+.Card__header.expanded {
+    .NextStepsView__cardHeader {
+        cursor: default;
+    }
+
+    .NextStepsView__cardHeaderBadge {
+        color: var(--button-color);
+        background-color: var(--sidebar-text-active-border);
+    }
+}
+
+.Card.complete {
```

Comment:
Does it make sense to have the complete state be attached to a generic card? If you use _something like_ `.Steps` at the accordion level and `.Steps_step` for each card, then the `.complete` state can be more appropriately attached.

Reply:
I'm not seeing how this is any more or less effective, but I think it goes back to our standing argument about the separation of behaviour between the `Card` and the `Accordion`.

I think for now I'd like to keep it as is and we can revisit it when we revisit that conversation.